### DISCUSSION TITLE
Mobile platforms not an option??

### DISCUSSION THREAD

**Discussion by jacobc94** [AvaloniaUI/Avalonia#15609] [Q&A] [answered]
Hi,
I am on a PC running Visual Studio 2022 17.9.6 and upon looking at the samples, I realized I could not run any other platforms except Windows. I was under the impression that I could be able to build and deploy Android and iOS as well as macOS and Linux. When looking at the new projects dialog, it seems like the only option for mobile is the "Cross Platform" project but we do not want WebAssembly. Can you please clarify?
Thank you.

**Comment by maxkatz6** [comment, marked as answer]
> it seems like the only option for mobile is the "Cross Platform" project but we do not want WebAssembly.

In the VS template builder, you can exclude WebAssembly.
If you create templates from the console (or simple vs templates), you simply can delete WebAssembly project, if you don't need it.

**Reply by jacobc94** [reply]
Thanks
